### ISSUE TITLE
Incorrect values in empty fields starting a row

### ISSUE THREAD

**Issue by arwer13** [vincentlaucsb/csv-parser#56]
Hello. It looks like if a row starts with an emtpy field that field and all the subsequent empty fields get initialized by the first non-empty field in the row.

E. g. parsing
```
category,subcategory,project name
,,foo-project
bar-category,,bar-project
```
gives
```
row 0
0 foo-project
1 foo-project
2 foo-project
row 1
0 bar-category
1 
2 bar-project
```

Example code used is
```
	std::string csvString(R"(category,subcategory,project name
,,foo-project
bar-category,,bar-project
	)");
	auto format = csv::CSVFormat();
	csv::CSVReader reader(format);
	reader.feed(csvString);
	reader.end_feed();
	auto rowNum = 0;
	for (auto row: reader) {
		qDebug() << "row" << rowNum;
		auto colNum = 0;
		for (auto col: row)	{fi
			qDebug() << colNum << col.get<>().c_str();
			colNum += 1;
		}
		rowNum += 1;
	}

	return 0;
```

**Comment by vincentlaucsb** [comment]
Thanks for reporting this! I'll look into it.

**Comment by vincentlaucsb** [comment]
Fixed by #57
